sphinx/ext/graphviz.py

```diff
@@ -284,7 +283,7 @@ def render_dot_html(self, node, code, options, prefix='graphviz',
                                    "'svg', but is %r") % format)
         fname, outfn = render_dot(self, code, options, format, prefix)
     except GraphvizError as exc:
-        logger.warning(__('dot code %r: %s'), code, text_type(exc))
+        logger.warning(__('dot code %r: %s'), code, str(exc))
```

Comment:
I think formatting with `%s` equals conversion with `str`. So this is duplicated. Is my understanding right?

Reply:
Yup! I agree. Done in #5814.